src/actions/initStore.ts

```diff
@@ -4,22 +4,33 @@ import { Connection } from '../Connection';
 import { sendTransaction } from './transactions';
 import { SetStore, Store } from '@metaplex-foundation/mpl-metaplex';
 
-interface IInitStoreParams {
+/** Parameters for {@link initStore} **/
+export interface InitStoreParams {
   connection: Connection;
+  /** Administrator wallet for the store **/
   wallet: Wallet;
+  /**
+   * - `true`: anyone can list on the store
+   * - `false`: only whitelisted creators can list on the store
+   **/
   isPublic?: boolean;
 }
 
-interface IInitStoreResponse {
+export interface InitStoreResponse {
   storeId: PublicKey;
   txId: string;
 }
 
+/**
+ * Initialize a {@link Store} account.
+ * This action will get a {@link Store} program derived account address for the provided wallet and initialize a store with that address, setting the given `wallet` as the admin
+ * @deprecated This action is deprecated, please use {@link initStoreV2} instead
```

Comment:
Is this deprecation correct? I don't see deprecation warnings in MPL or docs yet.

Reply:
My bad. I kind of made an assumption here, but realized later on that V2 is still WIP. I'll remove the deprecation notice until V2 is production ready. Should I somehow state V2 to be not ready yet?